gcloud/pubsub/topic.py

```diff
@@ -303,6 +303,31 @@ def set_iam_policy(self, policy, client=None):
             method='POST', path=path, data=resource)
         return Policy.from_api_repr(resp)
 
+    def test_iam_permissions(self, permissions, client=None):
```

Comment:
I'm a fan of `check_*` as it's more Pythonic.... We can always alias it if people become frustrated... I think the API method is the one that should change to `checkIamPermissions`